assets/js/modules/idea-hub/datastore/idea-state.js

```diff
@@ -32,15 +32,21 @@ const { receiveError, clearError } = errorStoreActions;
 
 const fetchPostUpdateIdeaStateStore = createFetchStore( {
 	baseName: 'updateIdeaState',
-	controlCallback: ( { name, saved, dismissed } ) => {
+	controlCallback: async ( { name, saved, dismissed } ) => {
```

Comment:
This `async` isn't needed here:
```suggestion
	controlCallback: ( { name, saved, dismissed } ) => {
```